app/src/main/java/org/gdg/frisbee/android/activity/AppInviteDeepLinkActivity.java

```diff
@@ -134,7 +133,7 @@ public void onConnectionSuspended(int i) {
     }
 
     @Override
-    public void onConnectionFailed(ConnectionResult connectionResult) {
+    public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
```

Comment:
Unfortunately lint warns me to do it because the parent implementation have that. We don't put @NonNull otherwise